File: Module3/3A(JavaScript)/JSCodingChallenge/challenge9.js
// Return the number of vowels in a string.

const string = "How many vowels are in this question?";

function countVowels(string) {
    
    string = string.toLowerCase();

    let count = 0;
    for (let char of string) {
        if (char === 'a' || char === 'e' || char === 'i' || char === 'o' || char === 'u') {
            count++;
        }
    }

    return count;
}

console.log("Number of vowels: "+ countVowels(string));

File: Module4/Tasks_4C/PythonCodingChallenge/challenge9.py
def count_vowels(string):
    string = string.lower()  # Convert the string to lowercase to handle case insensitivity
    vowels = "aeiou"  # Define a string containing all vowels

    count = 0
    for char in string:
        if char in vowels:
            count += 1

    return count

string = "How many vowels are in this question?"
print("Number of vowels:", count_vowels(string))
